bylaws/3-cf-council.md

```diff
@@ -103,6 +104,8 @@ Efficient participation and decision-making are essential for geographically dis
 - **Voting as a Last Resort:** If a consensus cannot be reached, the CFC resorts to voting. Decisions are then made by a simple majority of the voting CFC members present unless a supermajority is required.
 - **Supermajority Requirement:** Certain critical decisions require a supermajority vote. A supermajority is defined as at least two-thirds of the total number of CFC members.
 
+## Extended Governance Committee (EGC)
+
```

Comment:
I'd but a `Composition` section here, to be explicit about the fact that EGC = CFC + Project leaders, with equal votes. I know it's already in the preface, but this section should probably be self-sufficient?